lib/EnzymeCore/src/EnzymeCore.jl

```diff
@@ -267,6 +282,10 @@ const ReverseSplitWithPrimal = ReverseModeSplit{true, true, false, 0, true,Defau
 @inline set_runtime_activity(::ReverseModeSplit{ReturnPrimal,ReturnShadow,RuntimeActivity,Width,ModifiedBetween,ABI, ErrIfFuncWritten}, rt::Bool) where {ReturnPrimal,ReturnShadow,RuntimeActivity,Width,ModifiedBetween,ABI, ErrIfFuncWritten} = ReverseModeSplit{ReturnPrimal,ReturnShadow,rt,Width,ModifiedBetween,ABI, ErrIfFuncWritten}()
 @inline clear_runtime_activity(::ReverseModeSplit{ReturnPrimal,ReturnShadow,RuntimeActivity,Width,ModifiedBetween,ABI, ErrIfFuncWritten}) where {ReturnPrimal,ReturnShadow,RuntimeActivity,Width,ModifiedBetween,ABI, ErrIfFuncWritten} = ReverseModeSplit{ReturnPrimal,ReturnShadow,false,Width,ModifiedBetween,ABI, ErrIfFuncWritten}()
 
+@inline WithPrimal(::ReverseModeSplit{ReturnPrimal,ReturnShadow,RuntimeActivity,Width,ModifiedBetween,ABI, ErrIfFuncWritten}) where {ReturnPrimal,ReturnShadow,RuntimeActivity,Width,ModifiedBetween,ABI, ErrIfFuncWritten} = ReverseModeSplit{true,ReturnShadow,RuntimeActivity,Width,ModifiedBetween,ABI, ErrIfFuncWritten}()
```

Comment:
Should we make these lower case like the other setters?

Reply:
so we also have some setters uppercase at the moment as well https://github.com/EnzymeAD/Enzyme.jl/blob/9495698d2d1fa163413ed6a3e113a29a02292ea4/lib/EnzymeCore/src/EnzymeCore.jl#L261

I kind of like it uppercase like this since then WithPrimal(Forward) == ForwardWithPrimal and so on.

But yeah it is a shame it's not consistent (but it's already inconsnstent =/)

Reply:
I'm okay with either since we are already inconsistent. Let me know what you decide!

Reply:
I'm just going to go for this for now then. We can try to standardize more on the next bump